application/world/services/auto_bible_generator.py

```
@@ -448,13 +427,7 @@ async def _save_to_bible(self, novel_id: str, bible_data: Dict[str, Any]) -> Non
 
         # 先确保 Bible 记录存在
         try:
-            from domain.novel.value_objects.novel_id import NovelId
-            existing_bible = self.bible_service.bible_repository.get_by_novel_id(NovelId(novel_id))
-            if existing_bible is None:
-                # 创建 Bible 记录
-                bible_id = f"bible-{novel_id}"
-                self.bible_service.create_bible(bible_id=bible_id, novel_id=novel_id)
-                logger.info(f"Created Bible record for novel {novel_id}")
+            self._ensure_bible_exists(novel_id)
         except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)